### PR TITLE
added `graph` command

### PR DESCRIPTION
**Pull request by baljinderxd**
For #73

### CODE REVIEW

File: app/commands/graph.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const prepareAnswer = require('../utils/prepareAnswer');
+
+function graph(body, bot, chatId, n = 10) {
```
Comment (gurrrung) [inline app/commands/graph.js:3]: Since, we are passing `n`, how about supporting the N parameter? That would be cool, please.

**Review by gurrrung — CHANGES_REQUESTED** [review]
PTAL

File: app/commands/graph.js (hunk 1 above)
Comment (baljinderxd) [reply app/commands/graph.js:3]: Sure. I will add it too.

**Review by gurrrung — APPROVED** [review]
LGTM

**Pull request merged**